### DISCUSSION TITLE
Unirse a la organizacion launchX latam

### DISCUSSION THREAD

**Discussion by PabloLeonA** [Launch-X-Latam/MisionFrontEnd#17] [Q&A]
mision comander una pregunta el link para unirse a la organizacion no esta funcionando lo repararan en los proximos dias o como seria para unirse a la organizacion????

**Comment by Emilio17MC** [comment]
Hola!

Publicaron que por la cantidad de solicitudes alcanzaron el límite de invitaciones a la organización. Pero tan pronto esté disponible podrán seguir ingresando a la organización desde https://launchxauth.azurewebsites.net/

![image](https://user-images.githubusercontent.com/6946456/192406765-957fd28b-55a5-41ec-927d-13400352a332.png)
